xml/ay_bigfile.xml

```diff
@@ -14429,6 +14420,22 @@ echo -n 100
        <row>
         <entry>
          <para>
+          <literal>autoyast2</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Location of the control file for automatic installation,
+          see <xref linkend="Commandline.ay"/> for details. Similar to
```

Comment:
See the [rendering of Commandline.ay](https://www.suse.com/documentation/sles-12/singlehtml/book_autoyast/book_autoyast.html#Commandline.ay).